sessions/css-selectors/demo-end/css/styles.css

```
@@ -29,17 +13,13 @@ body {
 }
 
 h1 {
-  color: var(--color-nemo);
-  text-shadow: 3px 3px var(--color-granite);
+  color: orangered;
+  text-shadow: 3px 3px darkslategray;
```

```suggestion
  color: #ff4a11;
  text-shadow: 3px 3px #252629;
```
Please use the neuefische colors :). They are the same for every challenge. I don't see how the `:root` selector is an issue here, because it's not part of the demonstration. I'm not dying on this hill, though. 🤷‍♀️

---

👍🏼 Ah, that makes sense. I did not notice that they are used everywhere :) 
Yes, you are right, it is already part of the start version of the demo. But when adding styles during the demonstration we will use the CSS variables for colors and margin/padding. In previous sessions I had the feeling that this confuses the students. 

In general, my point is to either introduce the `:root` selector in this session or as part of the css-structure session. So we do not have to explain the same thing twice 🤷🏼‍♀️

---

I would also argue to introduce the `:root` selector in the CSS Structure session when talking about variables. I'm afraid that this would confuse students too much at this point.